util/chunk/chunk.go

```diff
@@ -457,67 +493,106 @@ func (c *Chunk) TruncateTo(numRows int) {
 
 // AppendNull appends a null value to the chunk.
 func (c *Chunk) AppendNull(colIdx int) {
+	if colIdx == 0 && c.sel != nil { // use column 0 as standard
```

Comment:
we can extract this logic to a method of `Chunk` to save two lines.

Reply:
fixed.